src/config/layers.js

```diff
@@ -421,12 +433,20 @@ export const parks = new TrafimageGeoServerWMSLayer({
   },
 });
 
-export const stuetzpunktBahnhoefe = new HandicapLayer({
+export const stuetzpunktBahnhoefe = new MapboxStyleLayer({
   name: 'ch.sbb.stuetzpunktbahnhoefe',
   key: 'ch.sbb.stuetzpunktbahnhoefe',
   visible: true,
+  mapboxLayer: handicapDataLayer,
+  styleLayersFilter: ({ id }) => /stuetzpunkt/.test(id),
+  queryRenderedLayersFilter: ({ id }) => /stuetzpunkt/.test(id),
+  styleLayer: {
+    id: 'ch.sbb.stuetzpunktbahnhoefe',
+    type: 'symbol',
```

Comment:
Do you need these 2?

Reply:
Yes, else mapbox gives errors in the console.